frameworks/hdfs/tests/test_auth.py

```diff
@@ -116,6 +116,7 @@ def kerberos(configure_security):
 @pytest.mark.dcos_min_version('1.10')
 @sdk_utils.dcos_ee_only
 @pytest.mark.smoke
+@pytest.mark.sanity
```

Comment:
Noticed this while in here as well. Given that non-helloworld smoke tests arent being run, this was effectively dead code.